charts/src/main/java/hu/ma/charts/line/LineChart.kt

```diff
@@ -167,6 +170,16 @@ fun LineChart(
                     onHorizontalDrag = { change, _ ->
                       if (change.position.x >= 0 && change.position.x <= maxWidth.toPx()) {
                         drillDownPoint = change.position.x
+                        if (updateOnDrag) {
+                          val xinterval = maxWidth.toPx() / maxNumberOfPointsOnX
+                          val snapToPoint =
+                            snapToPoints(xinterval, drillDownPoint ?: 0f, data.series)
+                          if (snapToPoint != null && snapToPoint.x != lastSnapToPointX) {
+                            drillDownPoint = (snapToPoint.x) * xinterval
```

Comment:
Is it correct to update drillDownPoint here? Will the drillDownLine not snap to values while dragging when you do this update?